db/migrate/20160310044144_add_doc_id_to_topic.rb

```diff
@@ -0,0 +1,26 @@
+class AddDocIdToTopic < ActiveRecord::Migration
+
+  def up
+    add_column :topics, :doc_id, :integer, default: 0
+    add_column :docs, :topics_count, :integer, default: 0
+    add_column :docs, :allow_comments, :boolean, default: true
+
+    if Forum.where(name: 'Doc comments').first.nil?
```

Comment:
Well, since the doc discussions are taking place on the actual doc show view, I don't want them showing up in the public community area, hence the need for a private forum or other way to classify them and filter them out.